docs/modules/ROOT/pages/type-definitions/schema-configuration/global-configuration.adoc

```diff
@@ -0,0 +1,50 @@
+[[schema-configuration-global-configuration]]
+= Global Configuration
+
+Through the schema configuration, it is possible to disable a type of operation for the whole type definitions and not on type by type basis.
```

Comment:
Uhm,  I feel that both the initial and the suggested sentences do not give the right idea of what this section is about. What we need to express is that the same directives described in the section `type-configuration.adoc` (apart from the deprecated @exclude), can be used directly at the schema to describe the same behavior described on that page but for the whole type definitions rather than type by type.
Saying  "but not individually" sounds to me like that is not possible to specify this behavior individually, but is exactly what we describe on the page `type-configuration.adoc`! It's more like "the same directives described on `type-configuration.adoc` can be applied to the schema to describe the same behavior but globally", of course with better wording than mine!

Hope it gives you more context!

Reply:
Like it!